js/repeater.js

```diff
@@ -96,8 +96,7 @@
 		});
 		this.$prevBtn.on('click.fu.repeater', $.proxy(this.previous, this));
 		this.$primaryPaging.find('.combobox').on('changed.fu.combobox', function (evt, data) {
```

Comment:
@futuremint This event gets fired twice in certain cases with combobox which then calls `pageInputChange` twice.